ts-tests/__tests__/balances.test.ts

```diff
@@ -9,54 +10,86 @@ import {
 const container = new MetaDContainer();
 const TEST_ACCOUNT = '0x1111111111111111111111111111111111111111';
 
-beforeAll(async () => {
-  await container.start();
-});
+const value = '0x200'; // 512, must be higher than ExistentialDeposit
+const gasPrice = '0x3B9ACA00'; // 1000000000
```

Comment:
Would prefer working with numbers instead of hex + comments

```suggestion
const value = BigNumber.from(512); // must be higher than ExistentialDeposit
const gasPrice = BigNumber.from(1_000_000_000);
```

```typescript
import { ethers, BigNumber } from 'ethers';
```

Reply:
@canonbrother we going with this?

Reply:
yep.. prev i need cater web3 test.. but its removed
https://github.com/DeFiCh/metachain/pull/66/commits/2e4c2c9f27d6b31bfaeebb7d2f9a8a4246887fc3